samples/with-k8s-ref/theia-next.yaml

```diff
@@ -0,0 +1,29 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: theia
+spec:
+  started: true
+  template:
+    projects:
+      - name: project
+        git:
+          remotes:
+            origin: "https://github.com/che-samples/web-nodejs-sample.git"
+    components:
+      - name: theia
+        plugin:
+          kubernetes:
+            name: theia-next
+            namespace: devworkspace-plugins
+      - name: terminal
+        plugin:
+          kubernetes:
+            name: machine-exec
+            namespace: devworkspace-plugins
+    commands:
+      - id: say-hello
+        exec:
+          component: plugin
```

Comment:
Seems it's not a valid devfile because command references non-existing component.

Reply:
That's what I get for being careless :)

Reply:
I squashed this fix into the commit that adds samples